software/profiling/Vampir.rst

```diff
@@ -1,11 +1,320 @@
+
+   Running Vampir on a login node is best suited for small trace files.
+
+
+.. image:: /images/vampir_login_gui.png
+   :align: left
+   :width: 280px
+   :height: 125px
+
+|
+|
+|
+|
+|
+|
+
+After logging onto Summit, execute the series of commands below:
+
+.. code::
+
+   $ module load vampir
+
+   $ vampir &
+
+From here you can load a file resident on the file system.
+
+.. image:: /images/vampir_splash_screen_file_selection.png
+   :align: left
+   :width: 300px
+   :height: 200px
+
+|
+|
+|
+|
+|
+|
+|
+|
+|
+
+
+
+
+Vampir Using Vampirserver
+=========================
+
+
+.. Tip::
+
+   This connection method is the better option for larger trace files
+
+
+.. image:: /images/vampir_login_node_connect_to_compute.png
+   :align: left
+   :width: 350px
+   :height: 125px
+
+|
+|
+|
+|
+|
+|
+
+After connecting to `Summit <https://docs.olcf.ornl.gov/systems/summit_user_guide.html#connecting>`_ you will need to load the Vampir module
+and start the :ref:`Vampirserver. `<vamps>`
+
+.. code::
+
+   $ module load vampir
+
+   $ vampirserver start -- -P <projectID> -w <walltime> -q <queue>
+
+   #Example: vampirserver start -- -P 123456 -w 60 -q debug
+
+.. _vampserpw:
+
+.. attention:: Successful Vampirserver startup message should appear in terminal window. You will need this information!
+
+ | [Test@login5.summit ~]$ vampirserver start  - - -P 123456 -w 60 -q debug
+ | Launching VampirServer...
+ | Submitting LSF batch job (this might take a while)...
+ | Warning: more than 1 task/rank assigned to a core
+ | VampirServer 9.11.1 OLCF (4626dba5)
+ | Licensed to ORNL
+ | Running 4 analysis processes... (abort with vampirserver stop 10102)
+ | User: Test
+ | Password: XXXXXXXXXXXX
+ | VampirServer <10102> listens on: h50n05:30040
+
+----------------------------------------------------------------------
+
+
+Launch the Vampir GUI
+---------------------
+.. code::
+
+   $ vampir &
+
+.. _vampauth:
+
+Once the GUI has opened, you will need to connect to the Vampirserver using the
+``Remote File`` option as shown below. **If there is a 'recent files' window open, select 'open other'.**
+Enter the ``node ID`` and the ``port`` number and press 'Connect'.
+
+
+.. image:: /images/vampir_open_remote_location.png
+   :align: left
+   :width: 800px
+   :height: 300px
+
+|
+|
+|
+|
+|
+|
+|
+|
+|
+|
+|
+|
+|
+
+When the server authentication window pops up, you will need to enter your ``USERID``
+& the :ref:`Vampirserver password <vampserpw>` that was printed on the terminal screen. Once authenticated, you will be able to navigate through the filesystem to your **.otf2** files
+
+.. image:: /images/vampir_server_auth.png
+   :align: left
+   :width: 400px
+   :height: 300px
+
+
+|
+|
+|
+|
+|
+|
+|
+|
+|
+|
+|
+|
+|
+|
+
+Vampir Tunneling to Vampirserver
+===========================================
+
+.. image:: /images/vampir_reverse_connect_gui_to_compute.png
+   :align: left
+   :width: 350px
+   :height: 125px
+
+|
+|
+|
+|
+|
+|
+
+.. Tip::
```

Comment:
Here, you put the Tip after the image, where above you put it before. This should be consistent in all 3 sections. I actually think I prefer it after the image. 

Also, Tip should be Note.